src/komponenter/header/common/brodsmulesti/Brodsmulesti.tsx

```diff
@@ -88,14 +88,14 @@ export const Brodsmulesti = (props: Props) => {
                     <li>
                         <button
                             aria-label={finnTekst('brodsmulesti-se-alle', language)}
-                            className={`${style.iconViewAll} lenke`}
+                            className={`${style.iconViewAll} `}
```

Comment:
Kan forenkles litt uten template string
```suggestion
className={style.iconViewAll}
```